client/ayon_substancedesigner/api/project_creation.py

```diff
@@ -62,6 +62,55 @@ def parse_graph_from_template(graph_name, project_template, template_filepath):
     return graph_element
 
 
+def parse_dependencies_from_template(template_filepath):
+    """Parse dependencies from Substance template file
+
+    Args:
+        template_filepath (str): Substance template filepath
+    """
+    dependencies = []
+    # Parse the template substance file
+    substance_tree = etree.parse(template_filepath)
+    substance_root = substance_tree.getroot()
+
+    for element in substance_root.find('.//dependencies'):
+        dependencies.append(element)
+    return dependencies
+
+
+def add_dependencies_from_template(dependencies, temp_package_filepath):
+    """Add dependencies to the temp package file
+
+    Args:
+        dependencies (list): all dependencies
```

Comment:
Can we add a type hint to what needs to be in this list. Is it `List[xml.etree.ElementTree.Element]`.